orm_lib/inc/drogon/orm/BaseBuilder.h

```diff
@@ -260,7 +260,7 @@ class BaseBuilder
                    EFn&& exceptCallback) noexcept
     {
         auto binder = *client << gen_sql(client->type());
-        for (const std::string& a : gen_args())
+        for (const std::string a : gen_args())
```

Comment:
I think using const reference is ok here.
![image](https://github.com/drogonframework/drogon/assets/21002501/462e056b-13d5-4f85-86ed-822d89f78f5c)

Reply:
Alright, thank you!